server/src/main/java/com/skillmagnet/Course/CourseController.java

```diff
@@ -71,6 +75,14 @@ public ResponseEntity<?> createCourse(@RequestBody Course course) {
         }
 
         courseRepository.save(course);
+
+        try {
+            CourseNode courseNode = new CourseNode(course.getId(), course.getTitle());
+            courseNodeRepository.save(courseNode);
+        } catch (Exception e) {
+            System.out.println("GraphDBError: Failed to create Course " + course.getId());
+        }
```

Comment:
:memo: I'm wrapping all of these in try/catches as I don't really want this to break anything existing. I think I'll use for final demonstration but honestly probably won't make this public just because all the connection stuff involved is a bit jank.

Reply:
Unless you can find a good way to set this up to be like a local in-memory graph database, figure out how to _easily_ run local neo4j db, or how to like put up a flag to disabled all graph stuff withotu affecting rest of app. It's definitely possible to do one of those things, I just don't want to put anymore time into it as I wanna focus on wiring up frontend now